neuroid-reactnative-sdk-types/src/types.d.ts

```diff
@@ -16,7 +16,15 @@ export interface NeuroIDClass {
     getUserID: () => Promise<string>;
     getSDKVersion: () => Promise<string>;
     getScreenName: () => Promise<string>;
+    enableLogging: (enable?: boolean) => Promise<void>;
 }
 export interface NeuroIDConfigOptions {
     usingReactNavigation: boolean;
 }
+export interface NeuroIDLogClass {
+    enableLogging: (enable?: boolean) => void;
+    log: (...message: String[]) => void;
+    d: (...message: String[]) => void;
+    i: (...message: String[]) => void;
+    e: (...message: String[]) => void;
```

Comment:
There is an option for warning (NIDLog.w) and verbose(.v) in the android sdk logs. We could include those here or later since we don't have any specific logs for these yet. Just wanted to mention it.

Reply:
Good callout, I think we can add them later if we find a use for them. TBH we probably won't be using a ton of logs on the RN side because most of it will be happening in the native modules